styles/style.css

```diff
@@ -0,0 +1,1143 @@
+@font-face {
+    font-family: "fondo-import";
+    src: url("../et-line-font/fonts/et-line.ttf");
+}
+
+* {
+    box-sizing: border-box;
+    scroll-behavior: smooth;
+}
+
+body {
+    scroll-behavior: smooth;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: sans-serif, "Roboto";
+}
+
+:root {
+    --spetialGreen: #36bcb4;
+    --white: #fff;
+    --anotherback: #f9f9f9;
+    --buttonVisited: #e3861d;
+    --navColor: #5c5e5e;
+    --StandarBlack: #101010;
+}
+
+
+/*--------------- header and Navbar Styles section --------------------*/
+
+.principalImages {
+    width: 100%;
+    height: 850px;
+    font-size: 1.2rem;
+    box-sizing: border-box;
+    color: var(--white);
+    background: url(../img/home/home-bg-2.jpg);
+    background-size: contain;
+    background-position: center center;
+    background-size: cover;
```

Comment:
you are repeating this background-size

Reply:
use shorthand when possible to avoid multiple attributes in the same class definition